src/plugins/rpd/rpdplugin.h

```diff
@@ -0,0 +1,86 @@
+#ifndef RPDPLUGIN_H
+#define RPDPLUGIN_H
+
+#include "rpd_global.h"
+
+#include "layer.h"
+#include "mapformat.h"
+#include "plugin.h"
+#include "tilesetformat.h"
+
+#include <QObject>
+
+namespace Tiled {
+class Map;
+}
+
+namespace Rpd {
+
+class RPDSHARED_EXPORT RpdPlugin : public Tiled::Plugin
+{
+    Q_OBJECT
+    Q_INTERFACES(Tiled::Plugin)
+    Q_PLUGIN_METADATA(IID "org.mapeditor.Plugin" FILE "plugin.json")
+
+public:
+    void initialize() override;
+};
+
+
+class RPDSHARED_EXPORT RpdMapFormat : public Tiled::WritableMapFormat
+{
+    Q_OBJECT
+    Q_INTERFACES(Tiled::MapFormat)
+
+public:
+    enum SubFormat {
+        Rpd
```

Comment:
`mSubFormat` is actually unused so this can be removed.

Reply:
done